WordPressKit/WordPressKit/BlogServiceRemoteREST+SiteSettings.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+extension BlogServiceRemoteREST {
+
+    /// Fetch all Timezones from the API, with special handling for 'Manual Offsets' and 'UTC' sections
+    ///
+    /// - Parameters:
+    ///     - success The block that will be executed on success.
+    ///     - failure The block that will be executed on failure.
+    public func fetchTimeZoneList(success: @escaping ([String: [String: String]]) -> Void, failure: @escaping (Error) -> Void) {
+        let endpoint = "timezones"
+        let path = self.path(forEndpoint: endpoint, withVersion: ._2_0)
+
+        wordPressComRestApi.GET(path!, parameters: nil, success: { (response, _) in
+            let dict = response as? [String: AnyObject]
+            let timezonesByContinentDict = dict?["timezones_by_continent"] as? [String: AnyObject]
+            var resultsDict: [String: [String: String]] = [:]
+            timezonesByContinentDict?.keys.forEach({ (continent) in
+                var newContinentDict: [String: String] = [:]
+                let continentArray = timezonesByContinentDict?[continent] as? [[String: String]]
+                continentArray?.forEach({ (labelValDict) in
+                    guard let labelString = labelValDict["label"],
```

Comment:
There's some duplication between the continent and manual offset parsing here – perhaps you could pull out a small helper or closure to produce what we need from the label / value dictionary the API gives us. Perhaps a `timezoneFromDictionary:` method that gives us a custom `Timezone` struct which has a `name` and `value`?